timetracker/entries/views.py

```diff
@@ -1,6 +1,20 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
 
-from .models import Entry
+from .models import Client, Entry, Project
```

Comment:
How do you import models from other apps then? `from otherapp import models as otherapp_models` ?